src/stylesheets/utility/_icons.scss

```diff
@@ -0,0 +1,40 @@
+.tag-list__icon,
+.tag-list__btn {
+  align-items: center;
+  background-color: color(yellow);
+  border-radius: 50%;
+  border: solid rem(4.5) color(white);
+  box-shadow: 0 0 0 rem(6) #FFFFFF90;
```

Comment:
Just a heads up that this 8 digit hex code format is supported in Firefox & Chrome, but not IE or Edge. The alternative with wider support is `rgba()`.